web/src/katex_server.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-console */
```

Comment:
This isn’t part of the browser frontend and doesn’t belong in `web/src`, which the ESLint configuration defines to be in the browser environment.

https://github.com/zulip/zulip/blob/d98580ab14452c838a58f389402bf9eca2e39582/.eslintrc.json#L239-L250

Not that I have a clear alternative to propose. Maybe we make a `web/server` or `web/backend` directory?

Reply:
`web/server`, `web/servers`, and `web/koa` all seem fine to me.